src/app/projects/add-edit-project/form-tab-2002/form-tab-2002.component.ts

```diff
@@ -78,11 +78,7 @@ export class FormTab2002Component implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
-    this.config.getRegions().takeUntil(this.ngUnsubscribe).subscribe(
-      (data) => {
-        this.regions = data;
-      }
-    );
+    this.regions = this.configService.lists.map(item => item.type === 'region');
```

Comment:
This should just get the regions list, since we already parsed it in the configService

Reply:
Nice catch! Fixed in #17dd743c